runtime/lua/vim/shared.lua

```diff
@@ -553,7 +552,6 @@ end
 ---@param t table List-like table
 ---@return table Flattened copy of the given list-like table
 function vim.tbl_flatten(t)
-  vim.deprecate('vim.tbl_flatten', 'vim.iter(…):flatten():totable()', '0.12')
```

Comment:
let's just change it to 0.13 . The docs shouldn't exist. The problems with vim.iter will be addressed in 0.11